packages/sales-api-service/src/services/permissions.service.js

```diff
@@ -63,7 +63,7 @@ export const calculateEndDateMoment = async ({ permitId, startDate }) => {
   const permit = await getReferenceDataForEntityAndId(Permit, permitId)
   const duration = moment.duration(`P${permit.durationMagnitude}${permit.durationDesignator.description}`)
   if (permit.durationMagnitude === 12 && permit.durationDesignator.description === 'M') {
-    return moment(startDate).add(duration).subtract(1, 'day').tz('Europe/London').endOf('day')
+    return moment(startDate).tz(SERVICE_LOCAL_TIME).add(duration).subtract(1, 'day').tz(SERVICE_LOCAL_TIME).endOf('day')
```

Comment:
I realise there's no test for the bug here, but the bug doesn't manifest when running tests - moment exhibits buggy behaviour of its' own (see https://github.com/moment/moment/issues/4743) that masks the bug that manifests once the web app is running. So all I can do is fix the bug :/